### PR TITLE
Rahul/ifl 2633 chainport add api interface

### PR DESCRIPTION
**Pull request by patnir**
## Summary

Adds chainport response types and request wrappers

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and label it with `breaking-change-rpc` or `breaking-change-sdk`.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish-cli/src/utils/chainport/requests.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  ChainportVerifiedToken,
+} from './types'
+
+export const fetchChainportTransactionStatus = async (networkId: number, hash: string) => {
+  const config = getConfig(networkId)
+  const url = `${config.endpoint}/api/port?base_tx_hash=${hash}&base_network_id=22`
+
+  const response: {
+    data: ChainportTransactionStatus
+  } = await axios(url)
```
Comment (hughy) [inline ironfish-cli/src/utils/chainport/requests.ts:19]: nit -- should we use this as `axios.get` for consistency?

File: ironfish-cli/src/utils/chainport/requests.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (hughy) [inline ironfish-cli/src/utils/chainport/requests.ts:1]: Could we add type annotations for the return types of these functions?

Comment (patnir) [reply ironfish-cli/src/utils/chainport/requests.ts:1]: Done!

File: ironfish-cli/src/utils/chainport/requests.ts (hunk 1 above)
Comment (patnir) [reply ironfish-cli/src/utils/chainport/requests.ts]: Done

**Review by hughy — APPROVED** [review]

**Pull request merged**